lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'menu'
+class Order
+  attr_reader :my_order
+
+  def initialize(menu_klass = Menu)
+    @my_order = {}
+    @menu = menu_klass.new
+  end
+
+  def display_menu
+    @menu.duplicate_menu
+  end
+
+  def select_meal(dish, quantity=1)
+    #Consider dropping next two lines into private validator method.
+    unless @menu.duplicate_menu.keys.include?(dish)
+      raise "Please select a dish from the menu."
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)